### DISCUSSION TITLE
IP问题

### DISCUSSION THREAD

**Discussion by lwei65** [mikumifa/cppTickerBuy#30] [提问专区]
![image](https://github.com/user-attachments/assets/3f0cf1a6-a277-490f-8442-989cb570b760)
感觉可能会ip限制吗？我把速度设置10000ms，其中显示未开票的是我换的流量，显示频繁是wifi。
前面测试把wifi的ip限制了？？

**Comment by mikumifa** [comment]
未知，cpp的风控目前不知道规律
